actors/builtin/reward/reward_state.go

```diff
@@ -54,23 +58,24 @@ func ConstructState(currRealizedPower abi.StoragePower) *State {
 // Used for update of internal state during null rounds
 func (st *State) updateToNextEpoch(currRealizedPower abi.StoragePower) {
 	st.Epoch++
-
-	cappedRealizedPower := big.Min(BaselinePowerAt(st.Epoch), currRealizedPower)
+	st.ThisEpochBaselinePower = BaselinePowerFromPrev(st.ThisEpochBaselinePower)
+	cappedRealizedPower := big.Min(st.ThisEpochBaselinePower, currRealizedPower)
 	st.CumsumRealized = big.Add(st.CumsumRealized, cappedRealizedPower)
 
 	for st.CumsumRealized.GreaterThan(st.CumsumBaseline) {
 		st.EffectiveNetworkTime++
-		st.CumsumBaseline = big.Add(st.CumsumBaseline, BaselinePowerAt(st.EffectiveNetworkTime))
+		st.EffectiveBaselinePower = BaselinePowerFromPrev(st.EffectiveBaselinePower)
```

Comment:
As my comment above, I don't expect to see this being mutated here, but do expect it to be mutated once outside this loop, corresponding to one epoch elapsing.

I then expect `st.ThisEpochBaselinePower` to be set back where it was originally, as `st.ThisEpochBaselinePower = st.EffectiveBaselinePower`.